### PR TITLE
add PostsItemTitle to recentDiscussion

### PR DESCRIPTION
**Pull request by Raemon**

### CODE REVIEW

**Comment by Raemon** [comment]
Tests:
– checked that a long post title doesn't wrap in Latest Posts
– checked that a long post does wrap in RecentDiscussion
– checked that the tooltip worked on RecentDiscussion's post title

File: packages/lesswrong/components/comments/recentDiscussionThread.jsx [hunk 1]
```diff
@@ -142,7 +142,7 @@ class RecentDiscussionThread extends PureComponent {
         <div className={classes.postItem}>
 
           <Link className={classes.title} to={Posts.getPageUrl(post)}>
-            {post.title}
+            <PostsItemTitle postItem2 wrap post={post} />
```
Comment (jimrandomh) [inline packages/lesswrong/components/comments/recentDiscussionThread.jsx]: If you want to do this, the `postItem2` prop definitely needs to be renamed. In any case, given that `PostsItemTitle` contains the hover-hack, has `PostsItem` in its name, and has breakpoints that only make sense in the context of `PostsItem2` in it, I think it's a bad idea to use it in new places where those don't apply.

Comment (Raemon) [reply packages/lesswrong/components/comments/recentDiscussionThread.jsx]: I think it's really important for PostsItemTitle to be re-usable, and to work in contexts where it wraps.

Comment (jimrandomh) [reply packages/lesswrong/components/comments/recentDiscussionThread.jsx]: Ok, I think making `PostsItemTitle` reusable makes sense. But, it does need to be accompanied by at least a rename, of the component (say, to `PostTitle`), and of the `postItem2` property (which is being used here for something else, I'm not sure what).

**Comment by Raemon** [comment]
I have made further updates.

**Pull request merged**